src/app/components/event-card.tsx

```diff
@@ -25,7 +26,7 @@ export function EventCard(props: EventCardProps): JSX.Element {
       <Stack p="base-loose" spacing="base-loose">
         <SpaceBetween position="relative">
           <Text fontSize={2} fontWeight={500}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
```

Comment:
This should be moved to the Text component. It is already a span tag.

Reply:
The text components make the test fail intermittently, the code above makes the test less flaky.

Reply:
I understand the problem, but I don't agree with this solution ...especially if it is just `less flaky`. We can't add an extra span tag everywhere we use a Text component with a test. Why are we using the Text component here then? What is its purpose?

Reply:
I tested this earlier to make sure there isn't some bug with `<Text/>` not forwarding the `data-testid` prop. It works as expected.

Let's try and isolate what case it is exactly where the tests are flakey. Feel free to ping us when you see this happen @timstackblock, but Fara's concerns are legitimate. `<Text />` renders a `<span>` tag, so here, we're creating unnecessary DOM `<span><span>text</span></span>`.

Reply:
We cannot use the text component or the test will fail

Reply:
@timstackblock take a look at https://github.com/hirosystems/stacks-wallet-web/pull/2207

I've investigated this to see why that happens, and it seems that it works fine.

Reply:
Thanks for doing this @kyranjamie 👍

Reply:
Thanks @kyranjamie this was helpful